core/domain/exp_domain.py

```diff
@@ -2176,6 +2176,52 @@ def _convert_states_v26_dict_to_v27_dict(cls, states_dict):
 
         return states_dict
 
+    @classmethod
+    def _convert_states_v27_dict_to_v28_dict(cls, states_dict):
+        """Converts from version 27 to 28. Version 28 adds content_id to
+        customization_arg for interactions which can have translations.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+        translatble_interactions = [
+            'DragAndDropSortInput', 'ItemSelectionInput', 'MultipleChoiceInput']
+        for state_dict in states_dict.values():
+            if state_dict['interaction']['id'] in translatble_interactions:
+                customization_args = state_dict[
+                    'interaction']['customization_args']
+                new_values = []
+                content_ids = []
+                for index, value in enumerate(
+                        customization_args['choices']['value']):
+                    content_id = 'interaction_' + str(index + 1).zfill(5)
```

Comment:
I don't remember seeing this construction before in the other content ID PRs. Do we do the zero-padding for our other IDs which include an index?

Also probably should write `'interaction_%d' % (...)`